Lib/subprocess.py

```diff
@@ -127,11 +127,13 @@ def stdout(self, value):
     import msvcrt
     import _winapi
     class STARTUPINFO:
-        dwFlags = 0
-        hStdInput = None
-        hStdOutput = None
-        hStdError = None
-        wShowWindow = 0
+        def __init__(self,dwFlags=0,hStdInput=None,hStdOutput=None,
```

Comment:
This needs to be implemented via keyword only arguments as the Terry suggested in the issue.